src/transformers/pipelines/image_classification.py

```diff
@@ -34,6 +34,22 @@ class ImageClassificationPipeline(Pipeline):
     Image classification pipeline using any `AutoModelForImageClassification`. This pipeline predicts the class of an
     image.
 
+    Example:
+
+    ```python
+    >>> from transformers import pipeline
+
+    >>> classifier = pipeline(model="microsoft/beit-base-patch16-224-pt22k-ft22k")
+    >>> potential_words = classifier("https://huggingface.co/datasets/Narsil/image_dummy/raw/main/parrots.png")
+
+    >>> from transformers.testing_utils import nested_simplify
+
+    >>> nested_simplify(potential_words)  # The scores might vary slightly across pytorch/tensorflow versions.
+    [{'score': 0.442, 'label': 'macaw'}, {'score': 0.088, 'label': 'popinjay'}, {'score': 0.075, 'label': 'parrot'}, {'score': 0.073, 'label': 'parodist, lampooner'}, {'score': 0.046, 'label': 'poll, poll_parrot'}]
```

Comment:
You don't need to use this, doctest will only compare the floats to the precision you write them. So the test will pass without using `nested_simplify`.

Reply:
Really ?? That's awesome, I can remove it in a lot of places then !